README.md

```diff
@@ -73,3 +92,52 @@ To test:
 ```
 bazel test //...
 ```
+
+### Building a subset of the workspace
+
+To build all targets in a subpackage, for example gz-common:
+
+```
+cd ~/gz
+bazel build //common/...
+```
+
+Gazebo components are generally kept in subdirectories, to select those targets:
+
+```
+cd ~/gz
+bazel build //common/graphics/...
+```
+
+To see a list of all available targets:
+
+```
+cd ~/gz
+bazel query //...
+```
+
+## Writing Bazel rules
+
+### Linting
+
+Use [buildifer](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md) to lint build files
+
+```
+cd ~/gz
+bazel-bin/bazel/lint/buildifier --all         # Reformat all Bazel files.
+bazel-bin/bazel/lint/buildifier //common/BUILD.bazel  # Only reformat one file.
+```
+
+If buildifier was not build with the workspace (unusual), but it via
```

Comment:
not sure what this means? `but` -> `build` ?

Reply:
Fixed, thanks.